src/client/notifications.ts

```diff
@@ -0,0 +1,17 @@
+const icon = '/images/logo-transparent.svg';
+
+const sendNotification = (options, route) => {
+    if (Notification.permission === 'default') {
+        Notification.requestPermission();
+    } else if (Notification.permission === 'granted') {
+        new Notification('PlayHex', { ...options, icon }).onclick = function() {
+            if (route) route();
+            focus(window);
+            this.close();
+        };
```

Comment:
I checked the error, it's not because `Notification` is not defined, but just the constructor throw an error on mobile.
- Just catch the error
- also `focus();` seems not taking argument

```suggestion
        try {
            new Notification('PlayHex', { ...options, icon }).onclick = function() {
                if (route) route();
                focus();
                this.close();
            };
        } catch (e) {
            // TODO check compatibility on mobile. https://stackoverflow.com/questions/29774836/failed-to-construct-notification-illegal-constructor
        }
```